drivers/clock_control/clock_stm32_ll_mp1.c

```diff
@@ -19,6 +21,8 @@ static inline int stm32_clock_control_on(struct device *dev,
 
 	ARG_UNUSED(dev);
 
+	z_stm32_hsem_lock(CFG_HW_RCC_SEMID);
+
```

Comment:
There are 2 register banks to enable/disable the clocks, one for the MCU, one for the MPU so no HWSEM needed in this file

Reply:
Done